openwisp_controller/config/base/vpn.py

```diff
@@ -760,6 +778,9 @@ def _auto_vxlan(self):
         """
         if not self.vpn._is_backend_type('vxlan') or self.vni:
             return
+        if self.vpn.config.get('vxlan', [{}])[0].get('vni', 0) != 0:
+            self.vni = self.vpn.config['vxlan'][0]['vni']
```

Comment:
are you sure we want to do this? What if the vni in the VPN server configuration is changed?